.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This file should be removed from the commit.  It should not be there because of the .gitignore - not sure why it is here.  Can you figure out how to remove it?